tests/qa_automation/qaset_post_patch_run.pm

```diff
@@ -0,0 +1,148 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2019 SUSE LLC
```

Comment:
As this is a new file please just use the current year(s), e.g. `2019-2020 SUSE LLC` or `2020 SUSE LLC` depending on when you worked on this.

Reply:
Fixed.